### DISCUSSION TITLE
Dumping compatibility questions

### DISCUSSION THREAD

**Discussion by Cephiros** [sanni/cartreader#615] [General Chat]
Eather than continuing to request compatability questions on unrelated threads, this is probably a better way to do it. To answer the various questions so far:

32x can be dumped as a genesis cart, but doesnt use 32x.txt, this will be fixed in the new version.

Sonic+knuckles, i have dumped it both alone, and+sonic3. They both work, but sram must be dumped off of sonic3 alone, as that is where s+k saves are stored along with sonic3. I have not tested sonic1, or 2.

Nes unlicensed: If the mapper is supported, i expect they can be dumped, i have personally dumped a number of tengen without issue, and i have dumped micromachines as well, it came out a byte or two off, so not completely but i expect that might be my copy.

**Comment by scrap-a** [comment]
About the Sonic & knuckles + Sonic3's save, it seems that the cartridge is judged to have no RAM because the Sonic & knuckles header (which has no RAM) is refered.
This was not taken care of during implementation. Sorry about that. I will fix it soon.

About Sonic1 and 2, those have no RAM, so there is no change.

**Reply by scrap-a** [reply]
I tried it, but dumping the save via Sonic & Knuckles was very unstable. I tried adjusting the access interval, which made the problem a little better but did not resolve it. To make matters worse, when the dumping failed, the original save data in sonic3 was broken.

Therefore, it is better to dump the save data from Sonic3.

**Comment by 7ank0v1c** [comment]
> 32x can be dumped as a genesis cart, but doesnt use 32x.txt, this will be fixed in the new version.

So I just received my copy of DOOM for the 32X (my first doom ever... I know I know... haha 😂 ) 

Anyways, it successfully dumped and the CRC32 is correct, however I had to manually check. Is it fixed already where the 32x.txt will be read?

**Reply by PsyK0p4T** [reply]
32x.txt is used, just make sure to have the file at the SD root, and cart reader up-to-date. Can you update to the current build (v12.0) and confirm that dumping your Doom is working ?

**Reply by 7ank0v1c** [reply]
Just re-checked after updating to v12.0. I can confirm that it now works. Thanks =D
